docs/ref/models/meta.txt

```diff
@@ -144,6 +148,40 @@ Retrieving all field instances of a model
          <django.db.models.fields.related.ManyToManyField: groups>,
          <django.db.models.fields.related.ManyToManyField: user_permissions>)
 
+    This method may only be used once all models are loaded (that is, after a
+    call to :func:`django.setup()` completes); if called too soon, it will
+    raise an ``AppRegistryError``.
+
+.. _meta-local-fields:
+
+Working with a model before all apps are loaded
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In rare advanced cases (for instance, in a
+:data:`~django.db.models.signals.class_prepared` signal handler), it may be
+necessary to introspect the fields on a model class before the app registry is
+fully populated with all models. In this case, it is only possible to
+introspect the fields locally defined on the model, not relation fields defined
+on other models pointing to this one. The ``local_fields`` attribute is a list
+of locally-defined fields, and can be used for this purpose:
```

Comment:
no comma
use double :: at the end of this line instead of `.. code-block:: python`